test_runner/regress/test_download_extensions.py

```diff
@@ -41,17 +41,14 @@ def neon_env_builder_local(
     return neon_env_builder
 
 
+@skip_on_postgres(PgVersion.V16, reason="TODO: PG16 extension building")
```

Comment:
@ololobus just noticing this in passing, seems like something that should be in compute team backlog  + we could link the issue here?

Reply:
Yes, thanks for the heads up https://github.com/neondatabase/cloud/issues/20050